File: tests/python/programs/fibonacci.py
# vybe-test: python/programs/fibonacci
# origin: languages/python/tests/python/test_programs.rs
# vybe-test-mode: compile

def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

for i in range(10):
    print(fib(i))

File: tests/fortran/programs/fibonacci.f90
! vybe-test: fortran/programs/fibonacci
! origin: languages/fortran/tests/fortran/test_programs.rs

program fib
    integer :: n, a, b, temp, i
    n = 10
    a = 0
    b = 1
    do i = 1, n
        temp = a + b
        a = b
        b = temp
    end do
    print *, a
end program fib
